app/controllers/application_controller.rb

```diff
@@ -1,11 +1,12 @@
 class ApplicationController < ActionController::Base
-  protect_from_forgery with: :exception
 
   before_filter :configure_permitted_parameters, if: :devise_controller?
 
   protected
 
   def configure_permitted_parameters
-    devise_parameter_sanitizer.for(:sign_up) << [:first_name, :last_name]
+    devise_parameter_sanitizer.for(:sign_up) << [
+      :first_name, :last_name, :profile_photo
+] 
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.<br>Trailing whitespace detected.